native/cocos/editor-support/spine-wasm/spine-wasm.cpp

```diff
@@ -11,61 +11,75 @@
 
 using namespace spine;
 
-static void logToConsole(const char* message) {
-    EM_ASM({
-        console.log(UTF8ToString($0));
-    }, message);
+namespace {
+static const int LOG_LEVEL_ERROR = 3;
+static const int LOG_LEVEL_WARN = 2;
+static const int LOG_LEVEL_INFO = 1;
```

Comment:
Doesn't need `static` keyword in anonymous namespace.

Reply:
fixed